### ISSUE TITLE
Dashboard made for InfluxQL and not FLUX ?

### ISSUE THREAD

**Issue by Seqway** [MrDrache333/iobroker-tankerkoenig-grafana-dashboard#1]
Just one question:

I imported this Dashboard and it works very nice BUT I think it is made for InfluxQL connection to database ?
Because if I import it to a database connected with FLUX it isn't working. 
Is that right ?

**Comment by MrDrache333** [comment]
That's right. 👍🏻 At the Moment only InfluxQL is supported. Maybe i will add a flux dashboard later. If you already did that you can create a pull request and i'll add it to this Repo.

**Comment by Seqway** [comment]
Okay good to know.
I am working already on it and still need to test some things and optimise.
I will fork your dashboard and change it accordingly to FLUX and then give you a PR okay ?

**Comment by MrDrache333** [comment]
Yes. Thanks 😃
